### PR TITLE
First attempt on vehicle dataset with a random forest classifier

### PR DESCRIPTION
**Pull request by Sidrah-Madiha**
This is the first attempt to classify vehicle.csv dataset with random forest classifier.
Nest step is to try to improve this classifer.
Then move to experiment with other classifiers, to see which archives the best accuracy.

### CODE REVIEW

**Comment by Sidrah-Madiha** [comment]
Typos in above comment, correct comment is:
This is the first attempt to classify vehicle.csv dataset with random forest classifier.
_Next_ step is to try to improve this classifer.
Then move to experiment with other classifiers, to see which _achieves_ the best accuracy.

**Comment by Sidrah-Madiha** [comment]
it was my first attempt for fixes #2

File: dev/Sidrah-Madiha/allcustommodules.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#     print(X[0])
+#     print(y[0])
+#     print(X.shape)
+#     print(y.shape)
+#     print('the data attributes columns')
+#     print(X[:5,:])
+#     print('The target variable: ')
+#     print(y[:5])
+#     Split dataset into training set and test set
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2 ,random_state = 21)
```
Comment (dzeber) [inline dev/Sidrah-Madiha/allcustommodules.py]: The `0.2` test proportion does not match the comment in the notebook. In general, how did decide on this test set size? I would be good to include a comment about this in the notebook.

File: dev/Sidrah-Madiha/vehicles_dataset_classifer_v1.ipynb [hunk 2]
```diff
@@ -0,0 +1,944 @@
...
+   "source": [
+    "sidra.create_confusion_matrix_class_report(y_test, y_pred)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Conclusion:\n",
+    "Overall I got 78 percent accuaracy which doesn't seem good, next step for today will be to first try to improve this model then I will experiment with other models to see comparative performance of other models on this dataset."
```
Comment (dzeber) [inline dev/Sidrah-Madiha/vehicles_dataset_classifer_v1.ipynb]: Looking at the confusion matrix and evaluation metric table, can you expand on your interpretation of this overall accuracy? I notice that the per-class accuracy scores are quite different across the classes.
